lib/mixlib/install/options.rb

```diff
@@ -25,17 +25,7 @@ class Install
     class Options
       class InvalidOptions < ArgumentError; end
 
-      attr_reader :options, :errors, :original_platform_version
-
-      SUPPORTED_ARCHITECTURES = %w{
-        i386
-        powerpc
-        ppc64
-        ppc64le
-        s390x
-        sparc
-        x86_64
-      }
```

Comment:
I could always be persuaded. But then why not constrain platforms or platform versions, etc. Let's just let the results do their thing here.